index.js

```diff
@@ -8,9 +8,6 @@ const { configSpcp, configMyInfo } = require('./lib/express')
 
 const PORT = process.env.MOCKPASS_PORT || process.env.PORT || 5156
 
-if (!process.env.SINGPASS_ASSERT_ENDPOINT && !process.env.CORPPASS_ASSERT_ENDPOINT) {
-  throw new Error('Either SINGPASS_ASSERT_ENDPOINT or CORPPASS_ASSERT_ENDPOINT must be set')
-}
```

Comment:
Rather than just ignoring the lack of those env vars, I think it would be helpful to the developer if some sensible logging is put in place

Reply:
Added a warning on using `PartnerId` as assertion endpoint.

Reply:
Added a warning on using PartnerId as assertion endpoint.